terraform/modules/rds/main.tf

```diff
@@ -0,0 +1,51 @@
+
+resource "aws_ssm_parameter" "db_host" {
+  name        = "WORDPRESS_DB_HOST"
+  description = "Database Host Paramater"
+  type        = "SecureString"
+  value       = "  " # This to be replaced by RDS Host Endpoint
```

Comment:
You can add it as variable as well 😁

Reply:
Hey @alanlima, actually this value is an output from RDS Host Endpoint.

Reply:
Hi @mfreitassm, as you guys are talking about it, here are my comments:

It's just a suggestion :)

You can create only one generic resource like:
```
resource "aws_ssm_parameter" "this" {
  name        = var.name
  description = var.description
  type        = var.type
  value       = var.value
  overwrite   = var.overwrite

  lifecycle {
    ignore_changes = [
      value
    ]
  }
}
```

so you can call the ssm-parameter resource to create as many module as you wnat: 

```
module "db_name" {
  source = "../modules/ssm-parameter"
  //Parameters
}

module "db_host" {
  source = "../modules/ssm-parameter"
  //Parameters
}
```
Also, instead of calling it "rds", we could call it "ssm-parameter" (terraform/module/ssm-parameter) and use this module to manage any sort of data configuration and secrets.

```
module "my_data_configuration" {
  source = "../modules/ssm-parameter"
  //Parameters
}

module "my_super_secret" {
  source = "../modules/ssm-parameter"
  //Parameters
}
```

Please, let me know what you think :)

Reply:
> Hey @alanlima, actually this value is an output from RDS Host Endpoint.

So, correct if I am wrong, but the RDS module will output this variable that should be passed by terraform via input variable in your module right? I don't think that it would be ideal to replace this value manually. It is fine for the sensitive variables

@fabioacp I don't mind those changes, but I would not complicate that far at this point. I can see the advantages of those changes, but I don't think that they are necessary for now. Maybe if we start to create more parameters we can come back and refactor this module.

Reply:
ok, np.